src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -0,0 +1,394 @@
+    'NGAppNRate': np.ones(12) * 0.05,  # Manure Spreading: Base Nitrogen Loss Rate # NOQA
+    'NGAppPRate': np.ones(12) * 0.07,  # Manure Spreading: Base Phosphorus Loss Rate # NOQA
+    'NGAppFCRate': np.ones(12) * 0.12,  # Manure Spreading: Base Fecal Coliform Loss Rate # NOQA
+    'NGPctSoilIncRate': np.zeros(12),  # Manure Spreading: % Of Manure Load Incorporated Into Soil # NOQA
+    'NGBarnNRate': np.ones(12) * 0.2,  # Barnyard/Confined Area: Base Nitrogen Loss Rate # NOQA
+    'NGBarnPRate': np.ones(12) * 0.2,  # Barnyard/Confined Area: Base Phosphorus Loss Rate # NOQA
+    'NGBarnFCRate': np.ones(12) * 0.12,  # Barnyard/Confined Area: Base Fecal Coliform Loss Rate # NOQA
+    'PctGrazing': np.array([0.02, 0.02, 0.10, 0.25, 0.50, 0.50,
+                            0.50, 0.50, 0.50, 0.40, 0.25, 0.10]),  # Grazing Land: % Of Time Spent Grazing # NOQA
+    'PctStreams': np.ones(12) * 0.05,  # Grazing Land: % Of Time Spent In Streams # NOQA
+    'GrazingNRate': np.ones(12) * 0.05,  # Grazing Land: Base Nitrogen Loss Rate # NOQA
+    'GrazingPRate': np.ones(12) * 0.07,  # Grazing Land: Base Phosphorus Loss Rate # NOQA
+    'GrazingFCRate': np.ones(12) * 0.12,  # Grazing Land: Base Fecal Coliform Loss Rate # NOQA
+    'GRPctManApp': np.array([0.01, 0.01, 0.10, 0.05, 0.05, 0.03,
+                             0.03, 0.03, 0.11, 0.06, 0.02, 0.02]),  # Manure Spreading: % Of Annual Load Applied To Crops/Pasture # NOQA
+    'GRAppNRate': np.ones(12) * 0.05,  # Manure Spreading: Base Nitrogen Loss Rate # NOQA
+    'GRAppPRate': np.ones(12) * 0.07,  # Manure Spreading: Base Phosphorus Loss Rate # NOQA
+    'GRAppFCRate': np.ones(12) * 0.12,  # Manure Spreading: Base Fecal Coliform Loss Rate # NOQA
+    'GRPctSoilIncRate': np.zeros(12),  # Manure Spreading: % Of Manure Load Incorporated Into Soil # NOQA
+    'GRBarnNRate': np.ones(12) * 0.2,  # Barnyard/Confined Area: Base Nitrogen Loss Rate # NOQA
+    'GRBarnPRate': np.ones(12) * 0.2,  # Barnyard/Confined Area: Base Phosphorus Loss Rate # NOQA
+    'GRBarnFCRate': np.ones(12) * 0.12,  # Barnyard/Confined Area: Base Fecal Coliform Loss Rate # NOQA
+    'ShedAreaDrainLake': 0,  # Percentage of watershed area that drains into a lake or wetlands: (0 - 1) # NOQA
+    'RetentNLake': 0.12,  # Lake Retention Rate: Nitrogen
+    'RetentPLake': 0.29,  # Lake Retention Rate: Phosphorus
+    'RetentSedLake': 0.84,  # Lake Retention Rate: Sediment
+    'AttenFlowDist': 0,  # Attenuation: Flow Distance (km)
+    'AttenFlowVel': 4,  # Attenuation: Flow Velocity (km/hr)
+    'AttenLossRateN': 0.287,  # Attenuation: Loss Rate: Nitrogen
+    'AttenLossRateP': 0.226,  # Attenuation: Loss Rate: Phosphorus
+    'AttenLossRateTSS': 0,  # Attenuation: Loss Rate: Total Suspended Solids
+    'AttenLossRatePath': 0,  # Attenuation: Loss Rate: Pathogens
+    'StreamFlowVolAdj': 1,  # Streamflow Volume Adjustment Factor
+}
```

Comment:
I would prefer if most of these existed in gwlf-e so we don't have to update these default values in 2 places. There are also values that are more or less static, which can be omitted (ex. NRur/NUrb).